cmd/mattermost/commands/config.go

```diff
@@ -102,3 +114,123 @@ func configSubpathCmdF(command *cobra.Command, args []string) error {
 
 	return nil
 }
+
+func configShowCmdF(command *cobra.Command, args []string) error {
+	app, err := InitDBCommandContextCobra(command)
+	if err != nil {
+		return err
+	}
+	defer app.Shutdown()
+
+	// Check to see that no arguments are given on the command line. If so,
+	// return an error
+	if len(args) != 0 {
+		return fmt.Errorf("no arguments should be given")
+	}
+
+	// set up the config object
+	// Note: app.Config() returns a pointer value so make appropriate changes
+	config := app.Config()
+
+	// get the filePath of the config file
```

Comment:
Yo don't need to read the file at all, this should be something like:
```go
config := app.Config()
prettyPrint(configToMap(*config))
```

Reply:
Should the `prettyPrint()` function do the printing or should It be returning a string which I can print in the command?